core/controllers/topics_and_skills_dashboard.py

```diff
@@ -238,21 +240,67 @@ def post(self):
 class NewTopicHandler(base.BaseHandler):
     """Creates a new topic."""
 
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {},
+        'POST': {
+            'name': {
+                'schema': {
+                    'type': 'basestring',
+                    'validators': [{
+                        'id': 'is_nonempty'
+                    }, {
+                        'id': 'has_length_at_most',
+                        'max_value': (
+                            android_validation_constants.MAX_CHARS_IN_TOPIC_NAME
+                        )
+                    }]
+                }
+            },
+            'url_fragment': constants.SCHEMA_FOR_TOPIC_URL_FRAGMENTS,
+            'description': {
+                'schema': {
+                    'type': 'basestring',
+                    'validators': [{
+                        'id': 'has_length_at_most',
+                        'max_value': (
+                            android_validation_constants
+                            .MAX_CHARS_IN_TOPIC_DESCRIPTION
+                        )
+                    }]
+                }
+            },
+            'filename': {
+                'schema': {
+                    'type': 'basestring',
+                    'validators': [{
+                        'id': 'is_valid_filename'
+                    }]
+                }
+            },
+            'thumbnailBgColor': {
+                'schema': {
+                    'type': 'basestring'
+                }
+            },
```

Comment:
Thinking about it doesn't color have a specific format?

Reply:
added the regex pattern for color